api/nodes/serializers.py

```diff
@@ -221,6 +221,7 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
     current_user_can_comment = ser.SerializerMethodField(help_text='Whether the current user is allowed to post comments')
     current_user_permissions = ser.SerializerMethodField(help_text='List of strings representing the permissions '
                                                                    'for the current user on this node.')
+    current_user_is_contributor = ser.SerializerMethodField(help_text='Whether the current user is a contributor on this node.')
```

Comment:
Talked with @sloria and this is potentially more than we'd want for list views, so please only include this field on the NodeDetailSerializer so it doesn't have to do this check for every node in a list. _Unless_ it's going to cause caching problems for us, but I presume we'd want to get the node info fresh every time we transition to a new node route anyways in case something changed and we wouldn't just use the info from the list that we'd gotten before. If that's not the case, then we might want to profile this to see how much strain it might cause or bundle up the check with the current_user_permissions check somehow.